### ISSUE TITLE
execve: No such file or directory/Signal 11 when using -vv

### ISSUE THREAD

**Issue by ammen99** [cms-dev/isolate#26]
I downloaded CMS, configured it, however when testing a problem all tests fail with Signal 11. Now, I narrowed the problem down to the isolate binary - I try to run:
```
isolate --init
isolate --run -- <path_to_static_binary>
```
I have verified that the statically linked binary is correct and that it runs without the isolate command. However, the second command above fails with the message:
`execve(<path_to_static_binary>): No such file or directory`
If I add the -vv flag, then the error becomes
`Caught fatal signal 11`

How can I debug this? I'm running it without cgroups and the kernel supports the necessary features. I'm using Fedora 26 if that matters.

**Comment by ammen99** [comment]
update: I'm very sorry for bothering you, it seems I haven't read the manual carefully. I moved the library to the box directory and now it works. closing

**Comment by ammen99** [comment]
Actually, the problem arises when using CMS. I debugged the problem as much as someone unfamiliar with CMS/isolate, I noticed a very strange behaviour:

I used the keep_sandbox option from CMS, so that I can access the sandbox directories. The cms generates a commands.log file, which contains the command line for calling the isolate binary. However, if I run this line(by for example `sh commands.log`) I get the signal 11(crash). Then, if I remove some options(or change them, for example make wall time from 3 to 2 seconds or similar) or if I add some dummy option like `-vv` or `-x 0` the command starts working as expected. When I revert the options to their initial state isolate starts crashing again. Note that adding these debug options "solve" the crash only when added manually to the file(if I make CMS append them it doesn't work). Can somebody point me in the right direction to debug this on my machine? I wrote to @stefano-maggiolo but on his computer the line that CMS generates works without problems, so I guess this is some bug in the isolate binary.
